shell/testing/BUILD.gn

```diff
@@ -36,3 +36,40 @@ executable("testing") {
     "//third_party/skia",
   ]
 }
+
+# A second tester binary with SUPPORT_FRACTIONAL_TRANSLATION enabled.
+executable("testing_fractional_translation") {
+  output_name = "flutter_tester_v2"
```

Comment:
Another nit: update the output_name too.

Reply:
lutter_tester_testing_factional_translation?

Reply:
done